### PR TITLE
Merge pull request #73 from gcivil-nyu-org/develop

### PR DESCRIPTION
**Pull request by ty2492tianfan**
cicd please

### CODE REVIEW

File: nyu_event/settings.py [hunk 1]
```diff
@@ -28,7 +28,7 @@
 DEBUG = True
 
 # ALLOWED_HOSTS = ['*','nyu-event-dev.us-east-1.elasticbeanstalk.com']
-ALLOWED_HOSTS = ["127.0.0.1", "postgres-dev.us-east-1.elasticbeanstalk.com"]
+ALLOWED_HOSTS = ["127.0.0.1", "main2.us-east-1.elasticbeanstalk.com"]
```
Comment (Vialor) [inline nyu_event/settings.py:31]: [IMHO] we can keep both "postgres-dev.us-east-1.elasticbeanstalk.com" and "main2.us-east-1.elasticbeanstalk.com"

Comment (ty2492tianfan) [reply nyu_event/settings.py:31]: ok

**Review by ty2492tianfan — COMMENTED** [review]
ok

**Review by ty2492tianfan — COMMENTED** [review]
cicd

**Review by Vialor — APPROVED** [review]

**Pull request merged**